src/views/campaigns/newCampaign.vue

```diff
@@ -3,12 +3,12 @@
     <div class="new-campaign-form">
       <h1> A form to create a new campaign would go here </h1>
     </div>
-    <router-link to='/campaigns'>Back Home</router-link>
+    <router-link to='/'>Back Home</router-link>
   </div>
 </template>
 
 <script>
 export default {
```

Comment:
It seems like here is where we make another call to both the blockchain and BE to create the new campaign on the blockchain and then send the info about the newly created campaign to the BE.

Reply:
Does this mean the FE will act as a go between BE and BC? Not a concern, more a curiosity. 
BC = blockchain

Reply:
Yeah, that is how it is seeming to come together as of now. If we're having problems we can always try to implement `Web3.py` to have BC interactions from the BE as well.